jetty-embedded-6.1/src/main/java/org/jboss/arquillian/container/jetty/embedded_6_1/JettyEmbeddedContainer.java

```diff
@@ -106,6 +109,10 @@
     @DeploymentScoped
     private InstanceProducer<WebAppContext> webAppContextProducer;
 
+    @Inject
+    @ApplicationScoped
+    private InstanceProducer<ServletContext> servletContextInstanceProducer;
```

Comment:
Changed, I was not sure what was the difference between both scopes in terms of extensions.